cmd/collector/main.go

```diff
@@ -83,10 +84,15 @@ func main() {
 	ch.Serve(listener)
 
 	r := mux.NewRouter()
-	apiHandler := app.NewAPIHandler(jaegerBatchesHandler, zipkinSpansHandler)
-	apiHandler.RegisterRoutes(r)
+	app.NewAPIHandler(jaegerBatchesHandler).RegisterRoutes(r)
 	httpPortStr := ":" + strconv.Itoa(*builder.CollectorHTTPPort)
 	recoveryHandler := recoveryhandler.NewRecoveryHandler(logger, true)
+
+	go zipkin.StartHttpAPI(logger, zipkinSpansHandler, recoveryHandler)
+
+	zipkinHandler := zipkin.NewAPIHandler(zipkinSpansHandler)
```

Comment:
hmmm, I'm not seeing much point of this. Given that we've swapped zipkin under a new endpoint, doesn't make sense for this to run on the old port.